posts/aleo-programs-deployment.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Aleo Programs in Testnet3
+
+Aleo empowers you to write and build private applications easily.
+
+As you read in the title, this guide is a walkthrough on how to deploy Aleo programs in Testnet3, this will be achieved using the Aleo SDK. We hope this blog post will help you to achieve this and discover all the potential of private applications.
+
+Before you can start deploying Aleo programs you must have the Aleo SDK installed and must have an Aleo Program created.
```

Comment:
the rust and aleo install can be assumed, but you probably want to include the command to get your program created here.